boards/deprecated.cmake

```diff
@@ -13,903 +13,9 @@
 # https://docs.zephyrproject.org/latest/develop/api/api_lifecycle.html#deprecated,
 # so these aliases are eventually removed
 
-set(96b_carbon_DEPRECATED
-    96b_carbon/stm32f401xe
-)
-set(96b_carbon_nrf51_DEPRECATED
-    96b_carbon/nrf51822
-)
-set(96b_meerkat96_DEPRECATED
-    96b_meerkat96/mcimx7d/m4
-)
-set(actinius_icarus_bee_ns_DEPRECATED
-    actinius_icarus_bee/nrf9160/ns
-)
-set(actinius_icarus_ns_DEPRECATED
-    actinius_icarus/nrf9160/ns
-)
-set(actinius_icarus_som_dk_ns_DEPRECATED
-    actinius_icarus_som_dk/nrf9160/ns
-)
-set(actinius_icarus_som_ns_DEPRECATED
-    actinius_icarus_som/nrf9160/ns
-)
-set(adafruit_feather_DEPRECATED
-    adafruit_feather_nrf52840/nrf52840
-)
```

Comment:
Should we keep this one? It was added after v3.7.0 (in 5e789c3ae0c7cac997d03ab73b778a334e5186a0 as a follow-up to 4e830ec19f5e7475b673816fc092f0c2a9352a3e)

Reply:
Added back :)